challenges/deploy-loderunner/README.md

```diff
@@ -0,0 +1,61 @@
+# Deploy Loderunner
+
+## Background
+
+To create a load test for our NGSA application, we need to run our end-to-end test tool (Loderunner) which generates https requests and sends them to our NGSA application. More about Loderunner can be found [here](#loderunner). 
+
+The Loderunner application needs to be deployed on our AKS cluster. Create a Kubernetes manifest file (yaml) to define and create the appropriate resource(s) needed to deploy Loderunner. The inputs needed to deploy Loderunner on our AKS cluster are shown below. Using these inputs, Loderunner will generate approximately 1 request per second when deployed. 
+
+```yaml
+
+container:
+    ghcr.io/retaildevcrews/ngsa-lr:beta
+name:
+    l8r-load-1
+namespace:
+    ngsa
+args:
+    -l  "1000", -r, -s https://< your subdomain name >.aks-sb.com, -f memory-benchmark.json
+
+```
+
+### Prerequisite
+
+This challenge depends on [Pulling from GitHub Container Registry](../github-container-registry/README.md) to enable the cluster to pull container images from `ghcr.io` 
+
+### Validate
+
+After have applied your yaml file you can check your loderunner pod logs validate that HTTP requests are being sent.
+
+### Bonus
+
+Modify the input arguments to have Loderunner (lr8) generate approximately 50 req/sec.
+
+## Resources
+- [K8 Objects](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/)
+- [Pods](https://kubernetes.io/docs/concepts/workloads/pods/)
+- [Static-pod](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/)
+- [K8 Cheatsheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#creating-objects)
+
+## Hints
+
+
+### Loderunner
```

Comment:
[non-blocking]
I recommend moving this section with "Running Loderunner from command line" and "Sample Interactive Loderunner Command" higher. That way the hack participant can do these first on the command line before trying to deploy it to the cluster. Maybe somewhere between "Background" and "Validate".

Reply:
I think moving the section higher makes sense if we want to have a sequence of execution when doing the challenge that is not how the other challenges look like, we intentionally kept those sections under Hints since they are not really part of the challenge. are they?  I am happy to move it around if you like.